Improve doc content (#5)
FreeRTOS/backoffAlgorithm: README.md

> ## backOffAlgorthm Library

Author: ```suggestion
## backOffAlgorithm Library
```

> This repository contains the backoffAlgorithm library, a utility library to calculate backoff period for network operation retries (like failed network connection with server) using an exponential back-off with jitter algorithm. The backoffAlgorithm library is distributed under the [MIT Open Source License](LICENSE).

Author: ```suggestion
This repository contains the backoffAlgorithm library, a utility library to calculate backoff period for network operation retries (like failed network connection with server) using an exponential backoff with jitter algorithm. The backoffAlgorithm library is distributed under the [MIT Open Source License](LICENSE).
```

> This library supports the "Full Jitter" algorithm for exponential back-off with jitter.

Author: ```suggestion
This library uses the "Full Jitter" strategy for the for exponential backoff with jitter algorithm.
```
